net-misc/netkit-rsh/files/patches/000_all_sectty.patch

```diff
@@ -0,0 +1,32 @@
+--- netkit-rsh-0.17-pre20000412/rexecd/rexecd.c.sectty	Sat Oct  2 17:50:52 1999
```

Comment:
There are a lot of patches. They already come to 39K on my machine (with the in-tree copy).

The limiter for individual patches is 20K but FILESDIR often grows beyond that: https://qa-reports.gentoo.org/output/filesdir-usage.txt.

Could you pop these in a tarball on e.g. your own git repo or similar? I can do this if you prefer (I suppose it makes review easier).

Reply:
Well, there's a catch: after this security fix gets accepted and the project gets off the last-rites list, I would really like to update our patchset to/against current Debian, which has other feature-improvements like adding a -p port flag. It's going to be *way* easier to do that and to make clear what has changed, if I'm dropping/modifying individual patches, vs a new opaque tarball that has no clear history from the last one.

If you don't think that'd be a problem, then sure, I can make this a new PATCHVER instead, and ++ again when doing the next set of changes.

Reply:
Got it. Let's stick with the status quo and remember to do that later.

Reply:
Oh, and you can add a commit to drop this from package.mask by the way!